docs/privateregistry.md

```diff
@@ -0,0 +1,23 @@
+## Private Docker Registry
```

Comment:
Add a link to this doc from the requirements section related to docker registry.